lib/core.js

```diff
@@ -26,6 +26,7 @@ const parseEnv = function(options, { checkPath }) {
   env.manifestPath = "";
   env.upstream = true;
   env.upstreamRegistry = "https://packages.unity.com";
+  env.testables = false;
```

Comment:
Then `core.js` may no longer aware of the testable parameter.

Reply:
Wouldn't it work anyway? We are setting it by default, and it's only being overwritten with the optional value, so it'll always be set.

Reply:
It "works", but it shouldn't be here. The env contains shared global options for all commands. If `testables` don't make sense for other commands it should not have the default value here, to void confusing or potential conflicting. 

One example is the `deps` command, which has a `--deep` option. The function visit it directly through `options.deep`, which is not part of the `env`.

https://github.com/openupm/openupm-cli/blob/08c3f9bc479b7e0f06b5c7807c245973d09a5e68/lib/cmd-deps.js#L10

Reply:
Would it be fine if I modify the `_add` method to accept a second optional parameter for testables?

Reply:
https://github.com/openupm/openupm-cli/blob/08c3f9bc479b7e0f06b5c7807c245973d09a5e68/lib/cmd-deps.js#L10

Maybe just parse it from the options parameter? Like what the _deps function does.